agent/logger/audit/entry_types.go

```diff
@@ -69,11 +72,16 @@ func (g *getCredentialsAuditLogEntryFields) string() string {
 
 func constructCommonAuditLogEntryFields(r request.LogRequest, httpResponseCode int) string {
 	httpRequest := r.Request
+	url := httpRequest.URL.Path
+	// V2CredentialsPath contains the credentials ID, which should not be logged
+	if strings.HasPrefix(url, credentials.V2CredentialsPath+"/") {
```

Comment:
Thanks for catching this. I think this is an opportunity to fortify the audit log unit tests as well. Right now it uses [a dummy URL](https://github.com/aws/amazon-ecs-agent/blob/master/agent/logger/audit/audit_log_test.go#L35). Can you extend it so that we can ensure that proper values are logged for both `v1` and `v2` requests?